projects/laji/src/app/shared-modules/document-viewer/document-annotation/document-annotation.component.ts

```diff
@@ -373,7 +369,11 @@ export class DocumentAnnotationComponent implements AfterViewInit, OnChanges, On
   }
 
   setIndexPagination() {
-    return this.result.findIndex(i => i.unit.unitId === this.highlight);
+    if (!this.identifying) {
+      return this.result.findIndex(i => (i.unit.unitId === this.highlight));
+    } else {
+      return this.result.findIndex(i => (i.fullURL === this.highlight));
```

Comment:
this part doesn't need to use `this.identifying` since this could be written as:
```
return this.result.findIndex(i => (i.fullURL === this.highlight || i.unit?.unitId === this.highlight))
```
ps the name of this method is also little misleading sine it's not setting anything but returning the index